raiden/network/proxies/channel_manager.py

```diff
@@ -68,6 +68,10 @@ def __init__(
             address_encoder(manager_address),
         )
 
+        CONTRACT_MANAGER.check_contract_version(
+            proxy.contract_version(),
+            CONTRACT_CHANNEL_MANAGER)
```

Comment:
Please follow the [coding style](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md#python), refer to the section `Breaking function calls when line is above 99 characters`.

Specifically, please use a new line for the closing parens.